cloud-service-providers/azure/promptflow/README.md

```diff
@@ -0,0 +1,189 @@
+# End to End LLM App development with Azure AI Studio, Prompt Flow and NIMs
+
+ ![tl:dr](./images/contoso-chat-nim.png)
+
+**Table Of Contents**
```

Comment:
Can we add a quick intro section that describes in a paragraph what Azure AI Studio is and what we will be doing here?

Reply:
Done, thanks